### PR TITLE
update readme with more info

### PR DESCRIPTION
**Pull request by TomasTomecek**
- link to roles
- formatting, grammar
- link to blog posts

### CODE REVIEW

**Review by lbarcziova — APPROVED** [review]
thanks!

File: README.md [hunk 1]
```diff
@@ -1,12 +1,20 @@
...
-  and closing the old ones
-- `.github/workflows/rotate_roles.yaml` - Github action that runs the script periodically (each Friday)
+1. `clone_roles.py` - script that does the cloning of the issues, assigning of the team members
+   and closing the old ones
+2. `.github/workflows/rotate_roles.yaml` - Github action that runs the script periodically (each Friday)
+
+You can read more about the process in these blog posts:
+
+- [How to run an open-source service?](https://medium.com/@laura.barcziova/how-to-run-an-open-source-service-fb3303240e69)
+- [Inception of rotating roles](https://medium.com/@laura.barcziova/inception-of-rotating-roles-9caf971b3096)
```
Comment (majamassarini) [inline README.md:19]: This link seems broken 🤔

Comment (TomasTomecek) [reply README.md:19]: image

:confused:

Comment (TomasTomecek) [reply README.md:19]: will wait with merging once their outage is over

File: README.md [hunk 2]
```diff
@@ -1,12 +1,20 @@
...
 The repository contains:
 
-- `clone_roles.py` - script that does the cloning of the issues, assigning of the team members
-  and closing the old ones
-- `.github/workflows/rotate_roles.yaml` - Github action that runs the script periodically (each Friday)
+1. `clone_roles.py` - script that does the cloning of the issues, assigning of the team members
+   and closing the old ones
+2. `.github/workflows/rotate_roles.yaml` - Github action that runs the script periodically (each Friday)
+
+You can read more about the process in these blog posts:
```
Comment (lachmanfrantisek) [inline README.md:16]:
```suggestion
You can read more about the process in the following blog post series:
```

File: README.md (hunk 1 above)
Comment (TomasTomecek) [reply README.md:19]: Maja, all links work now for me correctly.

**Pull request merged**